net/restic-rest-server/files/etc/init.d/rest-server

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+USE_PROCD=1
+
+PROG=/usr/bin/rest-server
+
+start_instance() {
+        local cfg="$1"
+	local var
```

Comment:
Looks like you're mixing tabs and spaces for indentation. I would suggest standardizing on tabs.